### PR TITLE
fix #43962 cazkolik.com

### LINKED ISSUE #43962: cazkolik.com
### Issue URL (Ads)
[https://www.cazkolik.com/JazzliGundem/106913/semsiyeli_kadin_insan_mutsuz_olduguna_sevinir_mi?.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.cazkolik.com%2FJazzliGundem%2F106913%2Fsemsiyeli_kadin_insan_mutsuz_olduguna_sevinir_mi%3F.html)
### Comment
Username: @PentagramTr
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.3.2
Browser: | Firefox
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Strip URLs from tracking parameters, Self-destructing third-party cookies (2880), Block WebRTC, Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#43962

### CODE REVIEW

**Review by BlazDT — CHANGES_REQUESTED** [review]
Please also add these rules:

||cazkolik.com/include/banner_goster.asp
||cazkolik.com/resimler/bannerler
cazkolik.com###gizlegoster


For social filter:
embed.radio.co###socialAnchor

**Comment by Sergey-Lyapin** [comment]
> Please also add these rules:
> 
> ||cazkolik.com/include/banner_goster.asp
> ||cazkolik.com/resimler/bannerler

these rules affect element #gizlegoster which lead to another pages of this site or related sites (f.e. cazfm.com), perhaps we should not block this element and hide other banners with cosmetic rules?

**Comment by BlazDT** [comment]
#gizlegoster is a popup, so for Annoyances filter popus.
banner_goster.asp are ad banners.

**Comment by Sergey-Lyapin** [comment]
so it seems that if filter for ads is on and annoyances filter is disabled, this popup looks like
Screenshot_popup

File: TurkishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -239,6 +239,8 @@ yabancidizi.org#$##rek-container { display: none!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/43170
 trabzonhaber24.com#$#.modal-backdrop { display: none !important; }
 trabzonhaber24.com#$#body { overflow: visible !important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/43962
+cazkolik.com#$##solust { position: static!important; }
```
Comment (Alex-302) [inline TurkishFilter/sections/general_extensions.txt]: why we need this rule?

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -92,6 +92,9 @@ kayserihaber38.com###reklamTepe
 kayserihaber38.com###kapatReklam
 kayserihaber38.com###logoSagi
 kayserihaber38.com###zzz
+cazkolik.com##.klasiktegizle[width="942"][cellspacing="0"]
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt]: image

File: TurkishFilter/sections/specific.txt [hunk 3]
```diff
@@ -92,6 +92,9 @@ kayserihaber38.com###reklamTepe
...
+cazkolik.com##td[colspan="3"][width="612"][style*="padding-top:5px"]
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt]: image

File: TurkishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (BlazDT) [reply TurkishFilter/sections/general_extensions.txt]: When the other items are hidden, without the rule the text of the radio are not in their boxes.

**Pull request merged**